website/docs/components/link/standalone/index.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Link::Standalone
+title: Link (Standalone)
 description: A link used in isolation and not as a part of surrounding body text.
```

Comment:
For this, I think the more human readable option would be "Standalone Link" with "Inline Link" for the other link type. Just my opinion though.

Reply:
@KristinLBradley I thought about that- but then they wouldn't be next to each other in the list, which seems more user-friendly. WDYT?

Reply:
I would make the changes in the same PR. I think they'll still be grouped next to each other based on how the form controls are showing up.

Reply:
@Dhaulagiri do you want to incorporate these changes into your existing PRs and I'll close these?

Reply:
My two cents: I read this is <name> <type>, so Link (standalone) makes cents (😉) to me. Categorically it's a link with a variant

Reply:
They would still be grouped under the "Link" category I would assume.